src/System.Net.Sockets/src/System/Net/Sockets/_ReceiveMessageOverlappedAsyncResult.Windows.cs

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
```

Comment:
Since this is going to become throw away code for Windows (we plan to implement TPL async over the high-perf *Async APIs) I'm not sure it's a good idea to use it for *NIX.